clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5423,7 +5423,7 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         DisableRangeRounding = true;
     }
     if (DisableRangeRounding || HasFPGA)
-      CmdArgs.push_back("-fsycl-disable-range-rounding");
+      CmdArgs.push_back("-fsycl-range-rounding=disable");
```

Comment:
I see no additional entries to pass along `-fsycl-range-rounding=force`.  Should it be added?  A corresponding test should be added as well if so.

Reply:
No I don't think so. The above checks are just saying range rounding should be disabled for FPGA or for `-O0`. I don't think there are any cases (apart from the standard way of providing the flag) where we need to force range rounding

Reply:
Oh, what I mean here is that if a user passes `-fsycl-range-rounding=<arg>` on the command line, I don't see any hooks to pass it along to the `-cc1` compilation step.  If the intended interface is `-Xclang -fsycl-range-rounding=<arg>` then we are all good.

Reply:
I've actually changed this so that it is also a plain driver option. Let me know that you think the added test case is enough